backend/tests/core/DatabaseHelper.cs

```diff
@@ -16,54 +18,30 @@ public static class DatabaseHelper
         /// Uses an InMemoryDatabase instead of relational.
         /// </summary>
         /// <param name="helper"></param>
+        /// <param name="dbName"></param>
         /// <param name="permission"></param>
         /// <param name="ensureDeleted"></param>
         /// <returns></returns>
-        public static PimsContext CreatePimsContext(this TestHelper helper, Permissions permission, bool ensureDeleted = false)
+        public static PimsContext CreatePimsContext(this TestHelper helper, string dbName, Permissions permission, bool ensureDeleted = false)
```

Comment:
Needed to support multi-threading, which requires the database name to be unique for each test.